spec/models/claim/base_claim_spec.rb

```diff
@@ -523,6 +523,39 @@ class MockSteppableClaim < Claim::BaseClaim
       specify { is_expected.to be_nil }
     end
   end
+
+  describe '#unread_messages_for' do
```

Comment:
my personal preference is for tests to start with a subject and then use that subject

```
subject(:call) { claim.unread_messages_for(user) }
```

and then call
```
is_expected.to ...
```